packages/gatsby/gatsby-node.js

```diff
@@ -29,6 +31,26 @@ exports.onCreateWebpackConfig = ({ plugins, getConfig, actions }) => {
     ],
   });
 
+  if (process.env.NODE_ENV === 'production') {
+    actions.setWebpackConfig({
+      plugins: [
+        new SentryWebpackPlugin({
+          include: 'public',
+          ignore: ['app-*', 'polyfill-*', 'framework-*', 'webpack-runtime-*'],
```

Comment:
Where do these come from? Can link link docs/code?

Reply:
Didn't find any resource that explains what these files do, so added some comments to know what these files are, which should be self-explanatory to know why they are not relevant assets to upload.